frontend/src/features/map_planning/layers/drawing/labels/EditableTextinput.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useRef } from 'react';
+import { Html } from 'react-konva-utils';
+
+function getStyle(color: string) {
+  const isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+  const baseStyle = {
+    width: 'fit-content',
+    height: '100%',
+    border: '1px solid black',
+    padding: '0px',
+    margin: '0px',
+    background: 'none',
+    outline: 'none',
+    color: color,
+    fontSize: '24px',
+    fontFamily: 'sans-serif',
+    resize: 'both',
+  };
+  if (isFirefox) {
+    return baseStyle;
+  }
+  return {
+    ...baseStyle,
+    margintop: '-4px',
+  };
+}
+
+type EditableTextInputProps = {
```

Comment:
Consider restructuring the props akin to 'SimpleFormInputProps', allowing the user to change every property of the containted textarea field.